packages/client/ui/Input/Input.tsx

```diff
@@ -0,0 +1,21 @@
+import * as React from 'react'
+
+import {twMerge} from 'tailwind-merge'
+
+export interface InputProps extends React.InputHTMLAttributes<HTMLInputElement> {}
+
+export const Input = React.forwardRef<HTMLInputElement, InputProps>(
+  ({className, type, ...props}, ref) => {
+    return (
+      <input
+        type={type}
+        className={twMerge(
+          'flex h-11 w-full rounded border-2 border-slate-300 bg-transparent px-2 py-1 text-sm font-semibold focus:outline-none focus-visible:border-sky-500 disabled:cursor-not-allowed disabled:opacity-50 data-[placeholder]:text-slate-600',
```

Comment:
-1 on focus, this new input is blue, but the rest of our inputs go from grey to black. Also, the font is bolder in this input vs others.

![Screenshot 2023-09-28 at 17 23 58](https://github.com/ParabolInc/parabol/assets/39854876/a7467fcb-5230-45b7-8310-63b74a63c5dd)



![Screenshot 2023-09-28 at 17 22 10](https://github.com/ParabolInc/parabol/assets/39854876/381cb4ae-4bf7-42de-9261-626f59b53371)

Reply:
I made a new input to be matched with new select and multi select components we introduced previously. Not sure if I need to change it to match our old components, tagging @acressall for this question

Reply:
The design team has been working add focus states, and they are all sky 500, so this is good!

Reply:
I think the weight is incorrect though, in the [designs](https://www.figma.com/file/Vc09AD3hxef2ypriihfK0T/Flow---ad-hoc-teams?type=design&node-id=61-3235&mode=design&t=lJr67VlUQFnsOFGL-4) the label is semibold, but the placeholder and input are regular

Reply:
Updated
<img width="692" alt="image" src="https://github.com/ParabolInc/parabol/assets/466991/113d2879-de9f-4ef4-a305-93b02cce5b55">

Reply:
Updated:

<img width="693" alt="image" src="https://github.com/ParabolInc/parabol/assets/466991/2c372e74-6a95-4d6f-bda1-7c986cb54bc8">